MovieChecker/MovieChecker/BoxOfficeCollectionCell.h

```diff
@@ -0,0 +1,15 @@
+//
+//  BoxOfficeCollectionCell.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/27/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface BoxOfficeCollectionCell : UICollectionViewCell
+
+- (void)setThumbnailImageFromURL:(NSURL *)url;
```

Comment:
It would probably be preferable to have something like a `displayMovie:` method that takes a model object and leaves the details of what exactly is being done with that movie object up to the cell.